components/sections/all_products/stories/all_products.stories.tsx

```diff
@@ -0,0 +1,1011 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import AppPromoComponent from "../index";
+import { Sections } from "types";
+
+const args = {
```

Comment:
I think for default component arguments this long should be defined in a common file. What do you think?